tests/ert3/console/integration/test_cli.py

```diff
@@ -359,6 +287,97 @@ def test_cli_run_presampled(tmpdir):
         for val in real_coeff.values():
             assert isinstance(val, float)
 
+    samples = {idx: [] for idx in ("a", "b", "c")}
```

Comment:
Could we move this code into a function similar to the `assert_export` and reuse it in the test below? Probably have to pass in some arguments, like expected distribution, mean and std; but could still work out 🤷

Reply:
Yes, that is a good idea. I will have a look :+1: